app/helpers/jobs_helper.rb

```diff
@@ -1,3 +1,23 @@
 module JobsHelper
 
+	def build_ganglia_link( host, start_seconds, report_type, node_num, size )
+    ganglia_uri = ""
+    OODClusters.each do |c|
+      if host.eql?(c.id.to_s)
```

Comment:
You can just do

```ruby
c = OODClusters[host]
if c && c.custom_allow?(:ganglia)
  ...
```

instead of

```ruby
OODClusters.each do |c|
  if host.eql?(c.id.to_s)
    if c.custom_allow?(:ganglia)
      ...
```